src/gui.js

```diff
@@ -7702,16 +7723,6 @@ ProjectDialogMorph.prototype.shareProject = function () {
                         this.buttons.fixLayout();
                         this.rerender();
                         this.ide.showMessage('shared.', 2);
-
-                        // Set the Shared URL if the project is currently open
-                        if (proj.projectname === ide.projectName) {
```

Comment:
This is removed b/c by definition, the project URL should already be present. (And thus there's no need to see if you are sharing a different project than the open one.)